### DISCUSSION TITLE
How to load at reboot the modified driver on Pi Zero 2W?

### DISCUSSION THREAD

**Discussion by jotonedev** [seemoo-lab/nexmon#591] [Q&A] [answered]
The current guide for the Raspberry Pi 3 is not applicable to the Zero 2W, because the brcmfmac.ko file is not present in the patch directory

**Comment by jlinktu** [comment, marked as answer]
The modified `brcmfmac` driver of different kernel versions are now located under sub-directories in `nexmon/patches/driver`.
